common/util.c

```diff
@@ -64,3 +68,36 @@ int get_modifier_names(const char **names, uint32_t modifier_masks) {
 
 	return length;
 }
+
+pid_t get_parent_pid(pid_t child) {
+	pid_t parent;
+	char file_name[100];
+	char *buffer = NULL;
+	char *token = NULL;
+	const char sep[2] = " ";
+	FILE *stat = NULL;
+
+	sway_log(L_DEBUG, "trying to get parent pid for child pid %d", child);
+
+	sprintf(file_name, "/proc/%d/stat", child);
+
+	if (!(stat = fopen(file_name, "r")) || !(buffer = read_line(stat))) {
+		return -1;
+	}
+
+	fclose(stat);
+
+	sway_log(L_DEBUG, "buffer string is %s", buffer);
```

Comment:
Yes, I actually already removed this on my local copy; I just forgot to do so on this prior to committing and pushing.